lib/serializer.js

```diff
@@ -379,7 +379,25 @@ function handleIncludes (resp, includes, relations, app) {
   })
 
   if (embedded.length) {
-    resp.included = _.flattenDeep(embedded)
+    // This array may contain duplicate models if the same item is referenced multiple times in 'data'
+    let duplicate = _.flattenDeep(embedded)
```

Comment:
Maybe change to var for now, we should probably drop support for node 0.10 but unsure about 0.12 so we may be stuck with var for now.